app/serializers/alchemy/json_api/element_serializer.rb

```diff
@@ -12,7 +12,7 @@ class ElementSerializer
       )
       belongs_to :parent_element, record_type: :element, serializer: self
 
-      belongs_to :page, record_type: :page, serializer: PageSerializer
+      belongs_to :page, record_type: :page, serializer: Alchemy::JsonApi::PageSerializer
```

Comment:
Maybe fully fully qualify the class ;)?

```suggestion
      belongs_to :page, record_type: :page, serializer: ::Alchemy::JsonApi::PageSerializer
```